builtin/bins/provider-azurerm/main_test.go

```diff
@@ -0,0 +1 @@
+package main
```

Comment:
It is - following the pattern of all other provider bins though. We should probably get rid of all of them.